src/restic/node.go

```diff
@@ -493,12 +559,35 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 	case "fifo":
 	case "socket":
 	default:
-		err = errors.Errorf("invalid node type %q", node.Type)
+		return errors.Errorf("invalid node type %q", node.Type)
+	}
+
+	if err = node.fillExtendedAttributes(path); err != nil {
+		return err
 	}
 
 	return err
 }
 
+func (node *Node) fillExtendedAttributes(path string) error {
+	if node.Type == "symlink" && node.LinkTarget == "invalid" {
```

Comment:
That's not correct, the ACL and the comment are not set for the symlink, but for the file the symlink points to:
```
# getfattr -d softlink-with-acl                                    
# file: softlink-with-acl
user.comment="this is a comment"

getfattr -d test
# file: test
user.comment="this is a comment"
```